### ISSUE TITLE
Symbol(s) not found for architecture in libStripe3DS2.a

### ISSUE THREAD

**Issue by MatthieuLemoine** [stripe/stripe-ios#1329]
## Summary
When building using Xcode 10.3 for simulator or device, the build fail with the following linking error:

```
Undefined symbols for architecture armv7:
  "_STDSSL_SSL_CTX_set_cert_cb", referenced from:
      _STDSSL_SSL_CTX_set_client_cert_cb in libStripe3DS2.a(ssl_x509.cc.o)
  "_STDSSL_SSL_SESSION_free", referenced from:
      _STDSSL_d2i_SSL_SESSION in libStripe3DS2.a(ssl_x509.cc.o)
      bssl::STDSSL::internal::DeleterImpl<ssl_session_st, void>::Free(ssl_session_st*) in libStripe3DS2.a(ssl_x509.cc.o)
  "_STDSSL_SSL_SESSION_to_bytes", referenced from:
      _STDSSL_i2d_SSL_SESSION_bio in libStripe3DS2.a(ssl_x509.cc.o)
  "_STDSSL_SSL_get_session", referenced from:
      _STDSSL_SSL_get_peer_certificate in libStripe3DS2.a(ssl_x509.cc.o)
      _STDSSL_SSL_get_peer_cert_chain in libStripe3DS2.a(ssl_x509.cc.o)
      _STDSSL_SSL_get_peer_full_cert_chain in libStripe3DS2.a(ssl_x509.cc.o)
      _STDSSL_SSL_get_verify_result in libStripe3DS2.a(ssl_x509.cc.o)
  "_STDSSL_SSL_use_PrivateKey", referenced from:
      do_client_cert_cb(ssl_st*, void*) in libStripe3DS2.a(ssl_x509.cc.o)
  "_STDSSL_X509_LOOKUP_file", referenced from:
      _STDSSL_X509_STORE_set_default_paths in libStripe3DS2.a(x509_d2.c.o)
      _STDSSL_X509_STORE_load_locations in libStripe3DS2.a(x509_d2.c.o)
  "_STDSSL_X509_LOOKUP_hash_dir", referenced from:
      _STDSSL_X509_STORE_set_default_paths in libStripe3DS2.a(x509_d2.c.o)
      _STDSSL_X509_STORE_load_locations in libStripe3DS2.a(x509_d2.c.o)
  "_STDSSL_i2d_SSL_SESSION", referenced from:
      pem_write_bio_SSL_SESSION_i2d(void const*, unsigned char**) in libStripe3DS2.a(ssl_x509.cc.o)
      pem_write_SSL_SESSION_i2d(void const*, unsigned char**) in libStripe3DS2.a(ssl_x509.cc.o)
     (maybe you meant: _STDSSL_i2d_SSL_SESSION_bio)
  "bssl::STDSSL::ssl_set_cert(bssl::STDSSL::CERT*, std::__1::unique_ptr<crypto_buffer_st, bssl::STDSSL::internal::Deleter<crypto_buffer_st> >)", referenced from:
      ssl_use_certificate(bssl::STDSSL::CERT*, x509_st*) in libStripe3DS2.a(ssl_x509.cc.o)
  "bssl::STDSSL::SSL_SESSION_parse(cbs_st*, bssl::STDSSL::SSL_X509_METHOD const*, crypto_buffer_pool_st*)", referenced from:
      _STDSSL_d2i_SSL_SESSION in libStripe3DS2.a(ssl_x509.cc.o)
  "bssl::STDSSL::ssl_has_certificate(bssl::STDSSL::SSL_HANDSHAKE const*)", referenced from:
      bssl::STDSSL::ssl_crypto_x509_ssl_auto_chain_if_needed(bssl::STDSSL::SSL_HANDSHAKE*) in libStripe3DS2.a(ssl_x509.cc.o)
      do_client_cert_cb(ssl_st*, void*) in libStripe3DS2.a(ssl_x509.cc.o)
ld: symbol(s) not found for architecture armv7
``` 

## iOS version
12.4

## Installation method
CocoaPod

## SDK version
16.0.6 & 16.0.7

## Other information
Could be linked to https://github.com/stripe/stripe-ios/issues/1266 & https://github.com/stripe/stripe-ios/issues/1288. But I'm already building with the latest iOS SDK & the latest Stripe SDK.

**Comment by davidme-stripe** [comment]
Hey @MatthieuLemoine, sorry that this isn't working for you! I have a branch that might fix it, can you add this to your Podfile?
`pod 'Stripe', :git => 'https://github.com/stripe/stripe-ios.git', :branch=>'davidme/add-more-symbols'`

Thanks!

**Comment by MatthieuLemoine** [comment]
@davidme-stripe It works using your branch. No more linking issue 🎉  Thanks! 
Will it be merged soon?

**Comment by davidme-stripe** [comment]
Thanks for confirming! We'll get it in a release very soon.

**Comment by davidme-stripe** [comment]
We shipped this in 17.0.0.
